cmd/proxima/proxima.go

```diff
@@ -0,0 +1,101 @@
+package main
```

Comment:
It's conventional to use main.go for the file which contains the main function and flags.